### ISSUE TITLE
Trouble shooting stationary mode on ios

### ISSUE THREAD

**Issue by edparsons** [christocracy/cordova-plugin-background-geolocation#186]
Hi Chris,

I'm using this plugin on an iphone 6 ios 8.3, the plugin works as expected until it goes into stationary mode then it never exits. I understand about the difference of the significant changes API, I have tested it many times going across the city so even the significant changes API should fire. I've also tried using xcode to simulate the device being in London and then San Francisco, which I would also expect the significant changes API to fire. Here is my config:

        bgGeo.configure(cb, function() {
          console.log('Background Location Error');
        }, {
            url: 'http://only.for.android.com/update_location.json', // <-- Android ONLY:  your server url to send locations to
            params: {
                auth_token: 'user_secret_auth_token',    //  <-- Android ONLY:  HTTP POST params sent to your server when persisting locations.
                foo: 'bar'                              //  <-- Android ONLY:  HTTP POST params sent to your server when persisting locations.
            },
            headers: {                                   // <-- Android ONLY:  Optional HTTP headers sent to your configured #url when persisting locations
                "X-Foo": "BAR"
            },
            desiredAccuracy: 10,
            stationaryRadius: 10,
            distanceFilter: 10,
            notificationTitle: 'Background tracking', // <-- android only, customize the title of the notification
            notificationText: 'ENABLED', // <-- android only, customize the text of the notification
            activityType: 'Fitness',
            debug: true, // <-- enable this hear sounds for background-geolocation life-cycle.
            stopOnTerminate: false // <-- enable this to clear background location settings when the app terminates
        });

Do you have any advise or steps I should check through to debug the problem. Are there any settings outside of the app like Background App Refresh that Apple need to have configured?

Thanks
Ed

**Comment by christocracy** [comment]
When you simulate location in simulator, use the [iOS Simulator->Debug->Location](https://www.dropbox.com/s/v0mz4wytf5o47bq/Screenshot%202015-05-25%2023.16.35.png?dl=0) menu -- Not Xcode->Debug->Location menu.

**Comment by christocracy** [comment]
Doing house-cleaning.  Is there an update on this?

**Comment by edparsons** [comment]
Hi Chris,

This turned out if you install other geo loc library they ask for
permission for only in app location on iOS and your plugin doesn't re-ask
for all the time permissions. So after hours of debugging through the obj-c
code, I realised all I needed to do was go into the permissions settings on
the phone and give the app always on geolocation permissions.

Might be worth a simple note in the documentation so others don't spend
similar hours after they've made such a simple mistake.

Ed
